controller/webhook.go

```diff
@@ -154,9 +157,14 @@ func (whsvr *WebhookServer) mutate(ctx context.Context, admissionReview *v1beta1
 
 	var patchOperations []PatchOperation
 
-	host, name, region := whsvr.getUpstreamEndpointParameters(nsLabels, &pod.ObjectMeta)
+	host, name, region, unsignedPayload := whsvr.getUpstreamEndpointParameters(nsLabels, &pod.ObjectMeta)
 
 	sidecarArgs := []string{"--name", name, "--region", region, "--host", host, "--port", ":8005"}
+	s, _ := strconv.ParseBool(unsignedPayload)
+	
+	if (s) {
+		sidecarArgs = []string{"--name", name, "--region", region, "--host", host, "--port", ":8005", "--unsigned-payload"}
```

Comment:
Similar to below, we should append: `sidecarArgs = append(sidecarArgs, "--unsigned-payload")`